src/MongoDB/Session.c

```diff
@@ -29,10 +29,10 @@
 
 zend_class_entry* php_phongo_session_ce;
 
-#define SESSION_CHECK_LIVELINESS(i,m) \
-	if (!(i)->client_session) { \
+#define SESSION_CHECK_LIVELINESS(i, m)                                                                                         \
+	if (!(i)->client_session) {                                                                                                \
 		phongo_throw_exception(PHONGO_ERROR_LOGIC TSRMLS_CC, "Cannot call '%s', as the session has already been ended.", (m)); \
-		return; \
+		return;                                                                                                                \
```

Comment:
Optional suggestion to break up the long line with `phongo_throw_exception()` so clang-format doesn't need to add so much hanging whitespace to all of the other lines.